### PR TITLE
ADR for the valid statuses of a project

### PR DESCRIPTION
**Pull request by JaymeeH**
- ref #637

### CODE REVIEW

File: spec/models/provenance_event_spec.rb [hunk 1]
```diff
@@ -2,25 +2,54 @@
...
+    # a status update event to mark is as pending
+    it "makes a provenance event recording the project submission" do
+      pe = described_class.new
+      pe.event_type = ProvenanceEvent::SUBMISSION_EVENT_TYPE
+      pe.event_details = "Requested by Joe Shmoe, 2023-01-19T12:00:00"
+      pe.event_person = "abc123"
+      pe.save
+      expect(pe.event_type).to eq(ProvenanceEvent::SUBMISSION_EVENT_TYPE)
+    end
+    it "makes a provenance event recording the status being set to pending" do
```
Comment (carolyncole) [inline spec/models/provenance_event_spec.rb:16]:
```suggestion
    it "makes a provenance event recording the status being set to approved" do
```

File: spec/models/provenance_event_spec.rb [hunk 2]
```diff
@@ -2,25 +2,54 @@
...
-    pe = described_class.new
-    pe.event_type = ProvenanceEvent::STATUS_UPDATE_EVENT_TYPE
-    pe.event_details = "The Status was updated from pending to approved"
-    pe.event_person = "abc123"
-    pe.save
+  context "when a project is made active" do
+    it "creates an active event type" do
+      pe = described_class.new
+      pe.event_type = ProvenanceEvent::ACTIVE_EVENT_TYPE
+      pe.event_details = "The Status was updated from pending to approved"
```
Comment (carolyncole) [inline spec/models/provenance_event_spec.rb:39]:
```suggestion
      pe.event_details = "The Status was updated from approved to active"
```

File: spec/models/provenance_event_spec.rb [hunk 3]
```diff
@@ -2,25 +2,54 @@
...
+      pe.event_person = "abc123"
+      pe.save
+      expect(pe.event_type).to eq(ProvenanceEvent::ACTIVE_EVENT_TYPE)
+    end
+  end
+  context "when a project status is updated" do
+    it "creates a status update event" do
+      pe = described_class.new
+      pe.event_type = ProvenanceEvent::STATUS_UPDATE_EVENT_TYPE
+      pe.event_details = "The Status was updated from pending to approved"
```
Comment (carolyncole) [inline spec/models/provenance_event_spec.rb:49]:
```suggestion
      pe.event_details = "The Status was updated from approved to active"
```

**Review by carolyncole — CHANGES_REQUESTED** [review]
Looks good.  Just a couple of small changes.

**Comment by bess** [comment]
@carolyncole Thank you, good catch! Fixed.

**Review by carolyncole — APPROVED** [review]

**Pull request merged**